Tasks/VsTest/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -2,13 +2,15 @@
   "loc.friendlyName": "Visual Studio Test",
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=624539)",
   "loc.description": "Run tests with Visual Studio test runner",
-  "loc.instanceNameFormat": "Test Assemblies $(testAssembly)",
+  "loc.instanceNameFormat": "Test Assemblies",
```

Comment:
discussed offline: there is a tracking bug for this.

---
In reply to: [87799232](https://github.com/Microsoft/vsts-tasks/pull/3065#discussion_r87799232) [](ancestors = 87799232)